toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -943,7 +958,8 @@ func hydrateFromLocalSource(fileHydrationState map[string]bool, newSourceDir str
 // Will alter `currentSignatures`.
 func hydrateFromRemoteSource(fileHydrationState map[string]bool, newSourceDir string, srcConfig sourceRetrievalConfiguration, skipSignatureHandling bool, currentSignatures map[string]string) {
 	const (
-		downloadRetryAttempts = 3
+		downloadRetryAttempts = 10
+		failureBackoffExp = 2.0
```

Comment:
Suggestion to use full words and add clarity:
```suggestion
		failureBackoffExponentBase = 2.0
```

Also, I'm not sure we need the `failure` prefix, but I have no strong opinion on this one.